dotnet/src/Connectors/Connectors.AI.HuggingFace/TextCompletion/HuggingFaceTextCompletion.cs

```diff
@@ -22,27 +22,27 @@ public sealed class HuggingFaceTextCompletion : ITextCompletion, IDisposable
     private const string HuggingFaceApiEndpoint = "https://api-inference.huggingface.co/models";
 
     private readonly string _model;
-    private readonly Uri _endpoint;
+    private readonly string? _endpoint;
     private readonly HttpClient _httpClient;
-    private readonly HttpClientHandler? _httpClientHandler;
+    private readonly bool _disposeHttpClient = true;
+    private readonly string? _apiKey;
 
     /// <summary>
     /// Initializes a new instance of the <see cref="HuggingFaceTextCompletion"/> class.
     /// </summary>
     /// <param name="endpoint">Endpoint for service API call.</param>
     /// <param name="model">Model to use for service API call.</param>
     /// <param name="httpClientHandler">Instance of <see cref="HttpClientHandler"/> to setup specific scenarios.</param>
+    [Obsolete("This constructor is deprecated and will be removed in one of the next SK SDK versions.")]
```

Comment:
Can we instruct on what they should do instead?

Reply:
Sure, done.